pyblox3/src/abtesting.py

```diff
@@ -6,16 +6,75 @@
 # Copyright © 2019- Sanjay-B(Sanjay Bhadra). All rights reserved.
 #
 
+
+# Info for sanjay, v1/enrollments is v1/get-enrollments, but the only difference is, is that SubjectType is a string
+# not an integer, but v1/enrollments can take an integer for SubjectType, as you can tell, this has enforced suffering
+# upon me. Also v1/get-enrollments can take strings for SubjectType, I believe 2 different engineers worked on these
+# endpoints, therefore they do the exact same thing, ABTestService didn't exist until 2018, so I can't see internal
+# code for it, my head hurts.
+
+# TODO Consider removing my existence
+# FIXME The awful code below.
+
 from .util import *
-import json
+
+root = "https://abtesting.roblox.com"
+
+
+# TODO Make this less verbose and use kwargs
+def createGetEnrollmentType(ExperimentName: str, SubjectType: str or int, SubjectTargetId: int) -> dict:
```

Comment:
It's a bit weird/uncanny that this is outside the class. Idk how I feel about that. Consider moving inside class.